### PR TITLE
Conform readme and docs

### PR DESCRIPTION
**Pull request by ikbalkaya**
This pull request aims to make Readme file conforming template described in this link
https://github.com/ably/ably-common/blob/main/templates/sdk-readme.md

Some notes 
* Documentation here has more sections here than in the standard template. I did not remove them as it made sense to have those additional sections (as there are subscribing / publishing SDKs)
* Sections are ordered with consideration of their semantics (For example I put resolution policy under "Usage" section as it was related)
* I renamed "Documentation" to "Resources" and changed "Useful resources" to "Useful links". I'm not sure if that's the correct approach but I thought it made sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +177,30 @@ For the greatest flexibility it is possible to provide a custom implementation o
 
 Please see `DefaultResolutionPolicy` [implementation](publishing-sdk/src/main/java/com/ably/tracking/publisher/DefaultResolutionPolicyFactory.kt) for an example.
 
+### Resources
+
+Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) documentation for a complete API reference and code examples.
+
+#### Useful links
```
Comment (QuintinWillison) [inline README.md:184]: For consistency, please add vertical whitespace below this line.

File: README.md [hunk 2]
```diff
@@ -199,6 +177,30 @@ For the greatest flexibility it is possible to provide a custom implementation o
...
+### Java Users
+
+We also provide support for applications written in Java, however the requirements differ in that case:
+- must wrap using the appropriate Java facade for the SDK they are using:
+    - [publishing-sdk-java](publishing-sdk-java/) for the [publishing-sdk](publishing-sdk/)
+    - [subscribing-sdk-java](subscribing-sdk-java/) for the [subscribing-sdk](subscribing-sdk/)
+- require Java 1.8 or later
+- require a minimum of Android API Level 24 at runtime
+
+
```
Comment (QuintinWillison) [inline README.md:203]: For consistency, please only ever use a single line of vertical whitespace.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ikbalkaya) [reply README.md]: https://github.com/ably/ably-asset-tracking-android/commit/dc5caa61414f94e61f2bed9d372c48acd9e8e67c

File: README.md (hunk 1 above)
Comment (ikbalkaya) [reply README.md:184]: https://github.com/ably/ably-asset-tracking-android/commit/dc5caa61414f94e61f2bed9d372c48acd9e8e67c

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**